--- my.cnf.orig	2015-01-18 21:37:25.415669371 +0000
+++ my.cnf	2015-01-18 21:55:04.132919266 +0000
@@ -61,8 +61,8 @@
 #
 # * Query Cache Configuration
 #
-query_cache_limit	= 1M
-query_cache_size        = 16M
+query_cache_limit	= 16M
+query_cache_size        = 128M
 #
 # * Logging and Replication
 #
@@ -93,6 +93,7 @@
 #
 # InnoDB is enabled by default with a 10MB datafile in /var/lib/mysql/.
 # Read the manual for more InnoDB related options. There are many!
+innodb_buffer_pool_size	= 256M
 #
 # * Security Features
 #
@@ -110,10 +111,11 @@
 [mysqldump]
 quick
 quote-names
-max_allowed_packet	= 16M
+max_allowed_packet	= 32M
 
 [mysql]
 #no-auto-rehash	# faster start of mysql but no tab completition
+join_buffer_size	= 64M
 
 [isamchk]
 key_buffer		= 16M
