--- php.ini.orig	2015-01-18 21:35:02.802962192 +0000
+++ php.ini	2015-01-18 21:54:43.324816084 +0000
@@ -660,7 +660,7 @@
 ; Its value may be 0 to disable the limit. It is ignored if POST data reading
 ; is disabled through enable_post_data_reading.
 ; http://php.net/post-max-size
-post_max_size = 8M
+post_max_size = 16M
 
 ; Automatically add files before PHP document.
 ; http://php.net/auto-prepend-file
@@ -788,7 +788,7 @@
 
 ; Maximum allowed size for uploaded files.
 ; http://php.net/upload-max-filesize
-upload_max_filesize = 2M
+upload_max_filesize = 16M
 
 ; Maximum number of files that can be uploaded via a single request
 max_file_uploads = 20
